config/rustworkx.sh

```diff
@@ -0,0 +1 @@
+MODULE_BUILD_DEPS="python-build-bundle rust"
```

Comment:
The `python-build-bundle` is automatically present, you can in fact remove this dep